static/src/javascripts/projects/common/modules/experiments/tests/contributions-story.js

```diff
@@ -0,0 +1,176 @@
+define([
+    'bean',
+    'qwery',
+    'common/utils/$',
+    'common/utils/template',
+    'common/views/svg',
+    'common/utils/fastdom-promise',
+    'common/utils/mediator',
+    'text!common/views/contributions-story.html',
+    'text!common/views/contributions-epic.html',
+    'common/utils/robust',
+    'inlineSvg!svgs/icon/arrow-right',
+    'common/utils/config',
+    'common/modules/commercial/commercial-features',
+    'common/utils/cookies',
+    'common/utils/element-inview'
+], function (bean,
+             qwery,
+             $,
+             template,
+             svg,
+             fastdom,
+             mediator,
+             contributionsStory,
+             contributionsEpic,
+             robust,
+             arrowRight,
+             config,
+             commercialFeatures,
+             cookies,
+             ElementInview
+) {
+
+    return function () {
+
+        this.id = 'ContributionsStory20160922';
+        this.start = '2016-09-22';
+        this.expiry = '2016-10-03';
+        this.author = 'Jonathan Rankin';
+        this.description = 'Test whether telling the story of the guardian through staggered messages over time results in more contributions than always showing the epic message.';
+        this.showForSensitive = false;
+        this.audience = 0.08;
+        this.audienceOffset = 0.48;
+        this.successMeasure = 'Impressions to number of contributions';
+        this.audienceCriteria = 'Non members viewing the UK edition';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'Telling the story of the guardian through staggered messages over time result in an uplift of 30% in impressions to conversions';
+        this.canRun = function () {
+            var worksWellWithPageTemplate = (config.page.contentType === 'Article'); // may render badly on other types
```

Comment:
The only other content type I can think of is the sponsored content (i.e. where a company pays for us to write content but it is editorially independent), I'm not sure who would be best placed to make that call though? It is editorial content instead of paid for but the funding comes from an external source?

Re: the Childrens Book Site, I'm not sure of why it was implemented on frontend but there's been slips with us having ads on these pages and it's been a big deal before. Because the page and content is written for children specifically it is meant to be the case that we never show any ads here (as you can see on the pages).

Happy to discuss further though @philwills ! Rich seems to know about the implementation from the NGW project from memory.